website/docs/index.html.markdown

```diff
@@ -0,0 +1,135 @@
+---
+layout: "vcfa"
+page_title: "Provider: VMware Cloud Foundation Automation"
+sidebar_current: "docs-vcfa-index"
+description: |-
+  The VMware Cloud Foundation Automation provider is used to interact with the resources supported by VMware Cloud Foundation Automation. The provider needs to be configured with the proper credentials before it can be used.
+---
+
+# VMware Cloud Foundation Automation Provider 1.0
+
+The VMware Cloud Foundation Automation provider is used to interact with the resources supported by VMware Cloud Foundation Automation. The provider needs to be configured with the proper credentials before it can be used.
+
+Use the navigation to the left to read about the available resources. Please refer to
+[CHANGELOG.md](https://github.com/vmware/terraform-provider-vcfa/blob/main/CHANGELOG.md)
+to track feature additions.
+
+## Supported VCFA Versions
+
+The following Cloud Director versions are supported by this provider:
+
+* 9.0
+
+## Connecting as System Administrator
+
+When you want to manage resources across different organizations from a single configuration.
+
+```hcl
+# Configure the VMware Cloud Foundation Automation Provider
+provider "vcfa" {
+  user                 = "administrator"
+  password             = var.vcfa_pass
+  auth_type            = "integrated"
+  org                  = "System"
+  url                  = var.vcfa_url
+  allow_unverified_ssl = var.vcfa_allow_unverified_ssl
+}
+
+# Create a new organization
+resource "vcfa_org" "org" {
```

Comment:
I'm think about making this whole example "copy-paste-able". What about:
- Adding the top `terraform { required_providers { ... } }` block (the issue is version, which would become stale in next release)
- Changing `vcfa_org` to the working `vcfa_tm_version` resource.

CC: @Didainius

Reply:
Yep, I initially thought the same, but as we would add more interesting resources next I didn't.

I have changed it, but I'd revisit it later to add these more-useful resources.